### PR TITLE
Bump dot-prop and use the new api

### PR DESCRIPTION
**Pull request by stevemao**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 'use strict';
-var dotProp = require('dot-prop');
+var dotPropGet = require('dot-prop').get;
```
Comment (stevemao) [inline index.js]: Is it a good name? :smile:

Comment (kevva) [inline index.js]: Keep `var dotProp = require('dot-prop');` and use `dotProp.get()` in the calls instead.

Comment (stevemao) [inline index.js]: Fine. Is it your coding style not to cache the used function directly? I thought it was slightly more performant?

Comment (kevva) [inline index.js]: Yeah, I guess there's a performance improvement to call it directly, so let us go with that. I guess the naming is ok too.

**Comment by stevemao** [comment]
:+1:

**Pull request merged**